wicket-util/src/main/java/org/apache/wicket/util/lang/Args.java

```diff
@@ -66,6 +66,26 @@
 		return argument;
 	}
 
+	/**
+	 * Checks argument is not empty (not null and has a non-whitespace character)
+	 *
+	 * @param argument
+	 *            the argument to check for emptiness
+	 * @param name
+	 *            the name to use in the error message
+	 * @return The {@code argument} parameter if not empty
+	 * @throws IllegalArgumentException
+	 *             when the passed {@code argument} is empty
+	 */
```

Comment:
please add a comment why there are two "identical" methods.
in few months/years someone will wonder what is going on

Reply:
Should I simply add: 

> This method overloads {@link #isEmpty(CharSequence)} for performance reasons.

Or should I speculate on why this is faster?

Reply:
`This method overloads {@link #isEmpty(CharSequence)} for performance reasons.` is enough.